rules/use-google-tag-manager/rule.md

```diff
@@ -44,4 +44,14 @@ Data and analytics result has recorded by Google Analytics on new SSW.Website. U
 
 ![Figure: GA4 is collecting data from the website](tracking-ga4.png)
 
-Learn more on [Google Tag Manager official site](https://marketingplatform.google.com/about/tag-manager/benefits/).
\ No newline at end of file
+Learn more on [Google Tag Manager official site](https://marketingplatform.google.com/about/tag-manager/benefits/).
+
+### Drawbacks
+While Google Tag Manager is great for non-tech people to easily embed tracking code, you must be careful to not introduce functionality to the website that may interfere with existing systems. 
+
+Some things to consider when adding GTM scripts to a container:
+
+* **Performance** - some scripts may have a large performance impact, and result in the slowing of the site. 
+* **Unexpected side effects** - scripts may interfere with existing systems on the site
+* **Duplication of code** - some functionality may already be implemented by developers within the site, which may cause a duplication of logic.
+* **Disconnected functionality** - some functionality may require developer modifications to ensure that it works correctly, and these customizations may enhance user experience.
```

Comment:
again, any examples? this is good to mention but at SSW we do have access to devs, so maybe we need an example.